onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -902,109 +911,109 @@ def main():
                 decoder_merged_model_fp32_opt_path,
             ]
 
-            # Run the optimizer script
-            logger.info("Optimizing models...")
-            for orig_path, opt_path in zip(old_paths, new_paths):
```

Comment:
All of this optimization code still needs to be called for the TorchScript exported model. Can you refactor this so that this optimizer code runs for the TorchScript exported model? You should also add the optimizer code that needs to run for the TorchDynamo exported model.